tests/std/include/range_algorithm_support.hpp

```diff
@@ -807,7 +807,7 @@ struct with_input_ranges {
     }
 };
 

```

Comment:
In today's episode of "Merge conflicts are fun!": the same removal of template default arguments needs to be made for `with_output_ranges` and `test_out` that are being added to this file today.

Reply:
Done.